back/engines/commercial/analysis/app/lib/analysis/LLM/claude_instant_1.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Analysis
+  module LLM
+    class ClaudeInstant1 < Base
+      def initialize(**params)
+        super
+        if !params.key? :region
+          ErrorReporter.report_msg('No AWS region specified for Claude Instant 1. Falling back to us-east-1.')
+          params[:region] ||= 'us-east-1'
```

Comment:
In order to not make mistakes and send any data to the wrong reason, maybe we should rather fail hard if the region is not provided?

Reply:
Sounds good to me :)